services/src/test/java/org/keycloak/utils/CookieHelperTest.java

```diff
@@ -32,4 +36,15 @@ public void testParseCookies() {
         
         Assert.assertEquals(3, values.size());
     }
+
+    @Test
```

Comment:
This test is not running any production code, therefore please remove this added test case.